composite/composite.go

```diff
@@ -48,9 +49,13 @@ func (c Compositor) Process(inputZipPath string, outputImagePath string) error {
 	return nil
 }
 
-func (c Compositor) savePng(image *image.NRGBA, outputPath string) {
+func (c Compositor) savePng(image image.Image, outputPath string) {
+	compressedImage := lossypng.Compress(image, lossypng.NoConversion, 30)
 	buf := new(bytes.Buffer)
-	png.Encode(buf, image)
+	enc := &png.Encoder{
+		CompressionLevel: png.BestCompression,
+	}
+	enc.Encode(buf, compressedImage)
```

Comment:
Are we sure we want to add a lossy compression here? Maybe we could try something like https://github.com/davidbyttow/govips to try and make the output size smaller? 

If you feel like this is out of scope you can create a new [in sprint] card for the work you have done here and move the file size card to the next sprint and tackle it in the next sprint?

Reply:
Looks like govips needs another binary installed on the machine and Windows support is "difficult". Not sure if we want that.

Reply:
For sure we don't want huge PNGs written by the non-optimal image/png library.

Reply:
Checked the documentation again and I see no way to link libvips to our binary. What do you think about using node.js with pkg: https://www.npmjs.com/package/pkg? @SxDx

Reply:
OK I think pkg is not really suitable and py2exe only produces Windows files. Will research more PNG libraries for golang for now.

Reply:
Rust would also be an option.